src/psyclone/gocean1p0.py

```diff
@@ -1788,8 +1831,14 @@ def grid_prop(self):
     @property
     def type(self):
         ''' The type of this argument - whether it is a scalar, a field or
-            a grid-property. The latter are special because they must be
-            supplied by the PSy layer. '''
+        a grid-property. The latter are special because they must be
+        supplied by the PSy layer.
+
+        :return; The type of this argument, either 'field', 'scalar', or \
```

Comment:
Thanks for updating this docstring. You have a typo in `:return:` though - second colon is currently a semi-colon.

Reply:
Done.